test/fixtures/coverage/functional_test/tool/dev.dart

```diff
@@ -0,0 +1,9 @@
+library tool.dev;
+
+import 'package:dart_dev/dart_dev.dart' show dev, config;
+
+main(List<String> args) async {
+  config.test.functionalTests = ['functional'];
```

Comment:
All tests should be in the `test/` directory, so instead of a directory structure like `project/functional/test/`, it should be `project/test/functional`. This fits with the other test suites as well:

```
project/test/unit/
project/test/integration/
project/test/functional/
```